src/research_etl/etl_sb_api/sb_api_job.py

```diff
@@ -0,0 +1,199 @@
+import os
+import datetime
+import tempfile
+
+from pyarrow import csv
+import pyarrow as pa
+import pyarrow.dataset as pd
+import pyarrow.compute as pc
+
+from research_etl.utils.util_rds import DatabaseManager
+from research_etl.utils.util_rds import copy_gzip_csv_to_db
+from research_etl.utils.util_logging import ProcessLogger
+from research_etl.etl_sb_api import sb_tables
+from research_etl.utils.parquet import ds_from_path
+from research_etl.utils.parquet import ds_metadata_min_max
+from research_etl.utils.parquet import ds_group_jobids
+
+
+SB_SCHEMA = "sb_api"
+BUCKET = os.getenv("SPRINGBOARD_BUCKET", "")
+SB_PREFIX = os.path.join(BUCKET, "odin", "data", "sb", "api", "")
+
+SB_TABLES = [
```

Comment:
So this controls the tables that will be ingested?

Reply:
correct, any ingested tables will also require that the primary tables be added manually to the DB. I'll add that information as a comment.